runtime/bcutil/ROMClassBuilder.cpp

```diff
@@ -457,13 +485,18 @@ ROMClassBuilder::prepareAndLaydown( BufferManager *bufferManager, ClassFileParse
 		return classFileOracle.getBuildResult();
 	}
 
-	SRPKeyProducer srpKeyProducer(&classFileOracle);
+	BuildResult injectRes = injectInterfaces(&classFileOracle);
```

Comment:
You can reuse the `BuildResult result = OK` above. In fact `BuildResult res = handleAnonClassName` is an error, there is no need to define multiple `BuildResult`